packages/calypso-products/src/plans-list.tsx

```diff
@@ -2134,6 +2144,26 @@ export const PLANS_LIST: Record< string, Plan | JetpackPlan | WPComPlan > = {
 		getPathSlug: () => 'ecommerce-2-years',
 	},
 
+	[ PLAN_WOOEXPRESS_MEDIUM_MONTHLY ]: {
+		...getPlanWooExpressMediumDetails(),
+		...getMonthlyTimeframe(),
+		availableFor: ( plan ) => [ PLAN_FREE, PLAN_ECOMMERCE_TRIAL_MONTHLY ].includes( plan ),
+		getProductId: () => 1053,
+		getStoreSlug: () => PLAN_WOOEXPRESS_MEDIUM_MONTHLY,
+		getPathSlug: () => 'wooexpress-medium-monthly',
+	},
+
+	[ PLAN_WOOEXPRESS_MEDIUM ]: {
+		...getPlanWooExpressMediumDetails(),
+		term: TERM_ANNUALLY,
+		getBillingTimeFrame: WPComGetBillingTimeframe,
+		availableFor: ( plan ) =>
+			[ PLAN_FREE, PLAN_WOOEXPRESS_MEDIUM_MONTHLY, PLAN_ECOMMERCE_TRIAL_MONTHLY ].includes( plan ),
+		getProductId: () => 1055,
+		getStoreSlug: () => PLAN_WOOEXPRESS_MEDIUM,
+		getPathSlug: () => 'wooexpress-medium',
```

Comment:
Assuming this is referring to the `store_product.product_slug`, I think it needs to be this:

```suggestion
		getPathSlug: () => 'wooexpress-medium-bundle-yearly',
```

Reply:
As above, this needs to be updated to either `wooexpress-medium-bundle-yearly` or `wooexpress-medium-yearly` to match either the `product_slug` or `path_slug` for the product. I think the latter will be better in general.

Reply:
Got, it. Thanks for clarifying.

Reply:
@nelsonec87 I've verified that the `store_product.extra` field has a `path_slug` which is in the format `wooexpress-<plan>-<interval>` for both the Small and Medium monthly and yearly products.